src/Compilers/CSharp/Test/Syntax/Parsing/ParsingErrorRecoveryTests.cs

```diff
@@ -5494,7 +5494,7 @@ public void TestModifiedMemberAfterPropertyGetBrace()
             Assert.False(pd.AccessorList.OpenBraceToken.IsMissing);
             Assert.NotNull(pd.AccessorList.CloseBraceToken);
             Assert.True(pd.AccessorList.CloseBraceToken.IsMissing);
-            Assert.Equal(2, pd.AccessorList.Accessors.Count);
+            Assert.Equal(1, pd.AccessorList.Accessors.Count);
```

Comment:
Error recovery got better here as well.  We understand that "public class d" is not an accessor.